<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QDialog" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>919</width>
    <height>387</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="customerButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Customer Info</string>
   </property>
  </widget>
  <widget class="QPushButton" name="inventoryButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>90</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>View Inventory</string>
   </property>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>0</y>
     <width>951</width>
     <height>831</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="customerPage">
    <widget class="QTableView" name="customerPage_tableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>562</width>
       <height>361</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="customerPage_addButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>10</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="NameLineEdit">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Name...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="MemberTypeLineEdit">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Member type...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ExpirationDateLineEdit">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>100</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Expiry Date...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="customerPage_deleteButton">
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>70</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="IDLineEdit">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>40</y>
       <width>111</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>ID number...</string>
     </property>
    </widget>
    <widget class="QLabel" name="companyLabel_2">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>220</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ’ª Bulk Club Inc.ðŸ’ª</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="inventoryPage">
    <widget class="QTableView" name="InventoryTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryPage_addButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>50</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryPage_editButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>90</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Edit</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ItemNameLineEdit">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>50</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Item name...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="QuantityLineEdit">
     <property name="geometry">
      <rect>
       <x>640</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Qty...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="InStockLineEdit">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>110</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Total Stock...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inventoryPage_deleteButton">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>130</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="ItemPriceLineEdit">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>80</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Price...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="RevenueLineEdit">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>140</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS UI Gothic</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Revenue...</string>
     </property>
    </widget>
    <widget class="QLabel" name="companyLabel_3">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>250</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ’ª Bulk Club Inc.ðŸ’ª</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="inventoryPage_searchBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter Item here...</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="membershipPage">
    <widget class="QPushButton" name="membershipPage_convertButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>161</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Convert Memberships</string>
     </property>
    </widget>
    <widget class="QTableView" name="MembershipTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="companyLabel_4">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>190</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Lucida Sans</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>ðŸ’ª Bulk Club Inc.ðŸ’ª</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="purchase">
    <widget class="QLineEdit" name="idLine">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>50</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Customer ID...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nameLine">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>80</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Item name...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="qtyLine">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>140</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Qty wanted...</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="priceLine">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>110</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Price...</string>
     </property>
    </widget>
    <widget class="QPushButton" name="confirmButton">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>200</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Make Purchase</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="dateLine">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>170</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>Date of purchase...</string>
     </property>
    </widget>
    <widget class="QTableView" name="testPurchase_inventoryTableView">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>561</width>
       <height>321</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="testPurchase_searchBar">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>10</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
      </font>
     </property>
     <property name="placeholderText">
      <string>Enter Item here...</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="companyLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="text">
    <string>ðŸ’ª Bulk Club Inc.ðŸ’ª</string>
   </property>
  </widget>
  <widget class="QPushButton" name="membershipButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Memberships</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans</family>
    </font>
   </property>
   <property name="text">
    <string>Test Purchase</string>
   </property>
  </widget>
  <widget class="QLabel" name="copyrightLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Â© COPYRIGHT 2020 Bulk Club Inc. All Rights Reserved.</string>
   </property>
  </widget>
  <widget class="QLabel" name="developerLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Lucida Sans Unicode</family>
     <pointsize>6</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Developed by The Simps.</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>confirmButton</sender>
   <signal>clicked()</signal>
   <receiver>Admin</receiver>
   <slot>test_purchase()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>770</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>386</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>determineUpgradeOrDowngrade()</slot>
  <slot>test_purchase()</slot>
  <slot>test_page()</slot>
 </slots>
</ui>
